src/main/java/polyglot/config/CommandLineArgs.java

```diff
@@ -50,6 +50,29 @@
   @Option(name = "--tls_ca_cert_path", metaVar = "<path>")
   private String tlsCaCertPath;
 
+  // *************************************************************************
+  // * Initial step towards the migration to "polyglot <command> [flagz...]" *
+  // *************************************************************************
+  @Option(name = "--command", metaVar = "<e.g. list_services>")	// Expand to support many commands
```

Comment:
Could we have the legal values in the metaVar (like for --with_message below)? Something like "list_services|call|